### PR TITLE
simplifying the test

### PR DESCRIPTION
**Pull request by DystopianProgrammer**

### CODE REVIEW

**Review by Mrdini — APPROVED** [review]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -1,30 +1,27 @@
 # Virgin Atlantic ~ Flight Information Display
-## Rules
-You can write the application in any language you see fit, bearing in mind that its sole intention is to showcase your
-skills for the applied-for position. 
 
-In addition you should adhere to the following conditions:
+Here's an example project for Virgin Atlantic Holidays, in a way that best represents what we are looking for in candidates for
```
Comment (spacebeers) [inline README.md:3]: Leave this is Virgin Atlantic. Take Vhols out of it as it's a bit confusing to read

File: README.md [hunk 2]
```diff
@@ -1,30 +1,27 @@
 # Virgin Atlantic ~ Flight Information Display
-## Rules
-You can write the application in any language you see fit, bearing in mind that its sole intention is to showcase your
-skills for the applied-for position. 
 
-In addition you should adhere to the following conditions:
+Here's an example project for Virgin Atlantic Holidays, in a way that best represents what we are looking for in candidates for 
+this role. Please spend no more than 2 hours improving this application. What you improve is up to you. Perhaps it needs more tests,
```
Comment (spacebeers) [inline README.md:4]: Is 2 enough? I would come back to this once you've tested it.

File: README.md [hunk 3]
```diff
@@ -1,30 +1,27 @@
 # Virgin Atlantic ~ Flight Information Display
-## Rules
-You can write the application in any language you see fit, bearing in mind that its sole intention is to showcase your
-skills for the applied-for position. 
 
-In addition you should adhere to the following conditions:
+Here's an example project for Virgin Atlantic Holidays, in a way that best represents what we are looking for in candidates for 
+this role. Please spend no more than 2 hours improving this application. What you improve is up to you. Perhaps it needs more tests, 
+or additional functionality. Let your imagination go wild and show us your best side!
```
Comment (spacebeers) [inline README.md:5]: This sentence is a bit wooly

File: README.md [hunk 4]
```diff
@@ -1,30 +1,27 @@
...
 
-The interface can be web-based, command-line or otherwise, but as already mentioned, you should choose the approach you
-feel demonstrates your expertise and suitability for the position.
-
 ## Supplying your code
 Please **create and commit your code into a public Github repository** and supply the link to the recruiter for review.  Your code should compile and run in one step.
 
 ## Supporting Data
-The [flight data](flights.csv) is a simple comma-separated file containing the following:
+The [flight data](Desktop/back-end-test/src/main/resources/flights.csv) is a simple comma-separated file containing the following:
```
Comment (spacebeers) [inline README.md:24]: Is this link correct?

**Pull request merged**